src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
Use `<img />` for site logo -  it is a part of the content. Otherwise use CSS `background-image`. 
Don't forget to add attributes: `width=""` and `height=""` for [performance](https://web.dev/articles/optimize-cls#images_without_dimensions) reasons. 

[Read more](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) about `<img />` and `background-image` differences